src/tribol/mesh/MfemData.hpp

```diff
@@ -601,6 +601,30 @@ class MfemMeshData {
    */
   void SetParentCoords( const mfem::ParGridFunction& current_coords );
 
+  /**
+   * @brief Sets a new reference coordinate grid function on the parent mesh
+   *
+   * @param reference_coords Reference coordinate grid function on the parent mesh
+   */
+  void SetParentReferenceCoords( const mfem::ParGridFunction& reference_coords );
```

Comment:
Make sure to unify the use of the word "position" vs "coords" throughout. My preference is "coords" in any api functions as is done here.

Reply:
will address in follow-on PR.  there is existing use of `Position` elsewhere.